front/src/config/i18n/en.json

```diff
@@ -917,7 +919,11 @@
           "friday": "Friday",
           "saturday": "Saturday",
           "sunday": "Sunday"
-        }
+        },
+        "house": "House"
+      },
+      "warning": {
+        "houseWithoutCoordinate": "Your house has no coordinate. This trigger cannot function without this data. Please go to Settings / Houses and click on the map to associate one."
```

Comment:
I would change the wording here : 

"Your house has no coordinate."  => "Your house has no coordinates.".
"This trigger cannot function without this data. " => "This trigger cannot work without this data. "

"Please go to Settings / Houses and click on the map to associate one." => "Please go to Settings / Houses and click on the map to set the house coordinates."